sd.config.js

```diff
@@ -50,7 +50,7 @@ export default {
     },
     js: {
       transformGroup: 'js',
-      buildPath: '/',
+      buildPath: '',

```

Comment:
why?

Reply:
Leading slash is not recommended in Backlight's style-dictionary config and will cause a non fatal error.

The reason for this is that when you run style-dictionary manually outside of Backlight, it will take this leading slash and try to execute on the root of your filesystem, which will throw an actual fatal error. So this is meant to make the config more compatible when you eject from Backlight